src/index.html

```diff
@@ -6,7 +6,10 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-  </head>
+      <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Useless space here. Please run `npx prettier --write .` to format the code

Reply:
Done !